packages/cozy-client/src/types.js

```diff
@@ -54,6 +54,7 @@ import { QueryDefinition } from './queries/dsl'
  * @property {string} id
  * @property {QueryDefinition} definition
  * @property {QueryFetchStatus} fetchStatus
+ * @property {boolean} isFetching
```

Comment:
I should also add directly here, the documentation, about isFetching (and take the time to document a little bit, the attribute of QueryState